src/app/core/components/directives/details-height/details-height.directive.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Directive, ElementRef, Inject, Input, OnDestroy, OnInit,
+} from '@angular/core';
+import { WINDOW } from 'app/helpers/window.helper';
+
+@Directive({
```

Comment:
Let's add a comment explaining briefly explaining why this is needed for future generations.